cmd/sriov/main.go

```diff
@@ -74,6 +74,23 @@ func cmdAdd(args *skel.CmdArgs) error {
 	defer netns.Close()
 
 	sm := sriov.NewSriovManager()
+	err = sm.FillOriginalVfInfo(netConf)
+	if err != nil {
+		return fmt.Errorf("failed to get original vf information: %v", err)
+	}
+	defer func() {
+		if err != nil {
```

Comment:
There is a call here
		if len(newResult.IPs) == 0 {
			return errors.New("IPAM plugin returned missing IP config")
		}
Where err wouldn't be != nil. Should we also handle that as well?

Reply:
Sorry, I might be misunderstanding,` if len(newResult.IPs) == 0 ` we are returning the error and it will not be equal to nil, defer function will release ipam and vf, and resetVFConfig. Is it wrong flow? Can you elaborate bit more what is the issue here?

Reply:
The defer function checks that "if err != nil" and in that specific case err will equal nil since it is a length check. So should we actually do something like this?
```
		if len(newResult.IPs) == 0 {
			err = ...
                        return err
		}
```

Reply:
Sorry I missed that. Now its fixed.

Reply:
looks good now, thanks!